### ISSUE TITLE
Annotation for Specifying Non null

### ISSUE THREAD

**Issue by cliedeman** [bpatters/schemagen-graphql#9]
Hello,

After implementing #7 I realised there is no easy way to define non null fields on GraphQL objects.

E.g. for RelayConnection pageInfo is a Non null field and currently the only way to enforce this would be to create a type mapper to RelayConnection.

The most natural solution is probably another annotation e.g. `GraphQLFieldNonNull` (To avoid collision with the graphql-core non null).

There is probably merit in also making any primitives non null by default. (This solves the pageInfo difference from #7  nicely)

Thoughts?

Ciaran

**Comment by bpatters** [comment]
Correct. Non-Null is something I hadn't gotten around to finishing. Shouldn't be difficult and it'd be nice to fully implement it by:
- Adding it for primitives
- Allow fields to be marked non-null with the annotation to modify the schema.

**Comment by cliedeman** [comment]
It seems that the annotation route may not be [possible](https://github.com/graphql-java/graphql-java-tools/issues/11#issuecomment-306582847)

Ciaran
